agent/consul/state/acl_schema.go

```diff
@@ -36,77 +35,77 @@ func tokensTableSchema() *memdb.TableSchema {
 				// DEPRECATED (ACL-Legacy-Compat) - we should not AllowMissing here once legacy compat is removed
 				AllowMissing: true,
 				Unique:       true,
-				Indexer: indexerSingle{
-					readIndex:  readIndex(indexFromUUIDString),
-					writeIndex: writeIndex(indexAccessorIDFromACLToken),
+				Indexer: indexerSingle[string, *structs.ACLToken]{
```

Comment:
It's a shame that the compiler cannot infer the type arguments here from `readIndex` and `writeIndex`'s types. Maybe in a future release? 🤞🏻